kernel/vmvalues.ml

```diff
@@ -62,7 +61,7 @@ type structured_constant =
 type reloc_table = (tag * int) array
 
 type annot_switch =
-   {ci : case_info; rtbl : reloc_table; tailcall : bool; max_stack_size : int}
+   { rtbl : reloc_table; tailcall : bool; max_stack_size : int }
```

Comment:
It seems worth adding a comment that points to `Is_tailrec_switch` in `coq_values.h`.